src/main/app/src/app/zaken/zaken-werkvoorraad/zaken-werkvoorraad.component.ts

```diff
@@ -105,11 +105,9 @@ export class ZakenWerkvoorraadComponent implements AfterViewInit, OnInit {
 
     assignToMe(zaakOverzicht: ZaakOverzicht, $event) {
         $event.stopPropagation();
-
-        this.zakenService.toekennenAanIngelogdeMedewerkerVanuitLijst(zaakOverzicht).subscribe(zaak => {
-            zaakOverzicht.behandelaar = zaak.behandelaar;
-            zaakOverzicht['behandelaar.naam'] = zaak.behandelaar.naam;
-            this.utilService.openSnackbar('msg.zaak.toegekend', {behandelaar: zaak.behandelaar.naam});
+        this.zakenService.toekennenAanIngelogdeMedewerkerVanuitLijst(zaakOverzicht).subscribe(() => {
+            zaakOverzicht['behandelaar'] = this.ingelogdeMedewerker;
+            this.utilService.openSnackbar('msg.zaak.toegekend', {behandelaar: this.ingelogdeMedewerker.naam});
         });
```

Comment:
Waarom was deze fix nodig? De code zoals hij was ziet er logisch uit. In de backend wordt de zaak toegekend aan de ingelogde gebruiker. De frontend neemt deze waarde daarna over. Waarom werkte dat niet?